applications/body_pose_estimation/CMakeLists.txt

```diff
@@ -22,3 +22,12 @@ add_custom_target(yolov8_onnx ALL
   COMMAND mv yolov8l-pose.onnx "${HOLOHUB_DATA_DIR}/body_pose_estimation"
   BYPRODUCTS "yolov8l-pose.onnx"
 )
+
+# Copy qos_profiles.xml from the dds_video application to the binary directory
+if(dds_video_subscriber IN_LIST HOLOHUB_BUILD_OPERATORS OR
+   dds_video_publisher IN_LIST HOLOHUB_BUILD_OPERATORS)
+  add_custom_target(body_pose_estimation_qos_profiles_xml ALL
+    COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/../dds_video/qos_profiles.xml" ${CMAKE_BINARY_DIR}
+    DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/../dds_video/qos_profiles.xml"
```

Comment:
The QoS profiles aren't actually tied to the operators, you can configure them however you want on a per application, domain, reader/writer, whatever basis -- there's a huge amount of flexibility when is comes to configuring your QoS settings.

However, in order for applications and readers/writers to be compatible with one another, they need to use compatible QoS settings. In the case of holohub applications, the expected use case is that the `body_pose_estimation` app will be used in conjunction with the `dds_video` app, and so they need to use compatible QoS settings. So while the `body_pose_estimation` app could provide its own `qos_profiles.xml` file, due to the expectation that it'll be used alongside `dds_video` I figured it was best to just reuse that configuration directly rather than duplicate it for the body pose app.

Reply:
Makes sense!